### PR TITLE
[PSR-6] Adding in missing singular deleteItem() and multiple saveItems()

### PR DESCRIPTION
**Pull request by dragoonis**

### CODE REVIEW

File: proposed/cache.md [hunk 1]
```diff
@@ -348,6 +359,17 @@ interface CacheItemPoolInterface
     public function save(CacheItemInterface $item);
 
     /**
+     * Persists multiple cache items immediately.
+     *
+     * @param array|\Traversable $items
+     *   A traversable collection or array of CacheItemInterface[] items
```
Comment (xabbuh) [inline proposed/cache.md]: Shouldn't this be `array of CacheItemInterface items`? Each is a cache item and not a collection of cache items.

Comment (dragoonis) [inline proposed/cache.md]: @xabbuh thanks for your feedback. The intent here is to say it's an array of CacheItemIterface elements, or a Traverable collection of CacheItermInterface items. As per the function above getItems(). Do you want to suggest an alternative description?

Comment (xabbuh) [inline proposed/cache.md]: I would keep it as is but simply remove the square brackets after the interface name:

> A traversable collection or array of CacheItemInterface items

**Comment by dragoonis** [comment]
@xabbuh updated, as per your feedback. Thanks.

**Pull request closed without merging**